src/com/duckduckgo/mobile/android/adapters/OnboardingAdapter.java

```diff
@@ -22,41 +17,26 @@
 
 public class OnboardingAdapter extends FragmentPagerAdapter {
 
-    private static final int NUM_PAGES = 5;
-    //private boolean useMiniLayout = false;
-    public OnboardingAdapter(FragmentManager fm) {
-        super(fm);
-        //NUM_PAGES = 5;
-        //Log.e("NUM_PAGES", "numPages: "+NUM_PAGES);
-    }
+    private final BaseOnboardingFragment[] items;
 
-    private OnboardingAdapter(FragmentManager fm, boolean useMiniLayout) {
+    public OnboardingAdapter(FragmentManager fm) {
         super(fm);
-        //this.useMiniLayout = useMiniLayout;
-        //NUM_PAGES = useMiniLayout ? 4 : 5;
-        //Log.e("NUM_PAGES", "numPages: "+NUM_PAGES);
+        items = new BaseOnboardingFragment[] {
```

Comment:
Great, feel free to grab me on slack if you run into any problems. I'm happy to pair program on this if you think it will help.